bd-api/src/api.rs

```diff
@@ -676,14 +642,24 @@ impl Api {
         // and process any additional frames we read together with the handshake.
         HandshakeResult::Received {
           stream_settings,
-          configuration_update_status: _configuration_update_status,
+          configuration_update_status,
           remaining_responses,
         } => {
           stream_state.initialize_stream_settings(stream_settings);
 
           self
             .handle_responses(remaining_responses, &mut stream_state)
             .await?;
+
+          // Let the all the configuration pipelines know that we are able to connect to the
+          // backend. We do this after we process responses that immediately followed the handshake
+          // in order to make it possible for the server to push down configuration that
+          // would get applied as part of the handshake response.
```

Comment:
Is the bit about processing the responses after the handshake still relevant?

Reply:
No let me fix that comment, thanks for catching.